README.md

```diff
@@ -4,7 +4,9 @@ LaunchDarkly SDK for .NET - Redis integration
 
 This library provides a Redis-backed persistence mechanism (feature store) for the [LaunchDarkly .NET SDK](https://github.com/launchdarkly/dotnet-client), replacing the default in-memory feature store. The underlying Redis client implementation is [StackExchange.Redis](https://github.com/StackExchange/StackExchange.Redis).
 
-The minimum version of the LaunchDarkly .NET SDK for use with this library is 5.2.1.
+The minimum version of the LaunchDarkly .NET SDK for use with this library is 5.6.0.
```

Comment:
Potentially put requirements like this in the changelog whenever they change

Reply:
I will do that when a release is happening, but it hasn't been our practice to update the changelog file before that time.